src/test/kotlin/com/igorwojda/integer/getodd/challenge.kt

```diff
@@ -4,7 +4,12 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun filterOdd(list: List<Int>): List<Int> {
-    TODO("not implemented")
+   return if (list.isNotEmpty())
+   {
```

Comment:
Recomandari codestyle:

- Bracket deschidere block se pune pe aceeasi linie cu if sau else: `if (cond) {` `else {` `else if (cond) {`
- Daca if are brackets, atunci si else are brackets, chit ca e 1 singura linie (si invers)
- Singurul caz in care if-else sunt onelinere sunt daca e 1 singura expresie in corp. Eg. `if (smth) { firstthing } else { scond }`